lesson02/assignment/css/style.scss

```diff
@@ -4,3 +4,57 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+$borderGray: #D9D9D9;
+$backgroundGray: #F2F2F2;
+$textColor: #595959;
+
+p {
+  color: $textColor;
+}
+.employee-card {
+  border: $borderGray solid 2rem;
+  width: 50%;
+}
+.card-content {
+  margin: 2rem;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-around;
+}
+.card-identity {
+  width: 30%;
+}
+.card-identity img {
```

Comment:
You mentioned not being sure where to use nesting, this is exactly where! You could combine this code with the code below it. Here's what this could look like:
`.card-identity {
img {
}
p {

}
}

Reply:
See a side by side of CSS and SCSS nesting here: https://sass-lang.com/guide